tools/parsinginjs/src/main/java/org/enso/parsinginjs/Patching.java

```diff
@@ -0,0 +1,37 @@
+package org.enso.parsinginjs;
+
+import org.apidesign.bck2brwsr.core.JavaScriptPrototype;
+
+class Patching {
+
+    private Patching() {
+    }
+
+    static void init0() throws Throwable {
+        try {
+            // loads the Statics class into the JVM
+            scala.runtime.Statics.avalanche(0);
+        } catch (Throwable t) {
+        }
+        try {
+            // replaces its class initialization and releaseFence methods
+            Statics.releaseFence();
```

Comment:
`releaseFence` is using _method handles_. Those aren't supported by [Bck2Brwsr VM](https://github.com/jtulach/bck2brwsr/). The `@JavaScriptPrototype` is a way to substitute such method with no-op.